lib/ip_subnet.go

```diff
@@ -64,3 +67,48 @@ func CIDRsFromIPRangeStrRaw(rStr string) ([]string, error) {
 
 	return r.ToCIDRs(), nil
 }
+
+// CIDRToIPSubnet converts a CIDR notation to IPSubnet.
+func CIDRToIPSubnet(cidr string) (IPSubnet, error) {
+	_, network, err := net.ParseCIDR(cidr)
+	if err != nil {
+		return IPSubnet{}, err
+	}
```

Comment:
We need to be getting these newlines/spacing right. It's used everywhere like this in Golang.

```suggestion
	}

```